common/v2/services/3Box/api.ts

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import Profile from './Profile';
+
+const API_ENDPOINT = 'https://3box.mycryptoapi.com/3box';
+
+export const get3BoxProfile = async (address: string): Promise<Profile | false> => {
+  const result = await axios({
+    method: 'GET',
+    url: `${API_ENDPOINT}?address=${address}`,
+    validateStatus: () => true
+  });
+
+  if (result.status === 200) {
+    const { data } = result;
+    return new Profile(address, data);
+  } else {
+    return false;
+  }
+};
```

Comment:
Prefer using `common/v2/services/ApiService/ApiService.ts`
We want to use a single instance of axios for all our ajax calls. 
It will become useful when we implement offline handling and retries.
(happy to discuss about this choice)